packages/terra-form/src/Textarea.scss

```diff
@@ -1,14 +1,87 @@
 :local {
   .textarea {
-    border: #dedfe0 solid 1px;
-    border-radius: 0.25em;
-    font-size: 1em;
-    line-height: 1.25;
-    padding: 0.178em;
+    background-color: var(--terra-form-textarea-background-color, #fff);
```

Comment:
Let's drop the `form` part in the variable and go with `--terra-textarea-` as the prefix.

Reply:
In the war room, we talked about having the individual form packages as -terra-form-select. Given that, I think we should be consistent with our namespacing.

Reply:
That's a good point, I'm okay with keeping this as is then.